src/products/firefox_android/how_to_localize.md

```diff
@@ -2,16 +2,16 @@
 
 Firefox for Android is on a four week rapid release cycle. This means that usually there is a new release going out every four weeks.
 
-Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that localizers can complete their work. The deadline in Pontoon is an indicator of when the next localization cut-off date is for the upcoming release - in other words, the last day to get localization updates into that release.
+Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that localizers can complete their work. The deadline in [Pontoon](https://pontoon.mozilla.org/projects/firefox-for-android/) is an indicator of when the next localization cut-off date is for the upcoming release - in other words, the last day to get localization updates into that release.
 
-## Localize Firefox for Android in Pontoon
+Note that all files contained in the project should be localized in order for Firefox for Android to be complete.
```

Comment:
What about android-components?

Reply:
That's what I mean by "all files contained in project", since the android-component strings are in the project, and all files there need to be localized. Example: https://pontoon.mozilla.org/yo/firefox-for-android/

Reply:
Oh, that's the part I'm missing. I didn't realize Firefox for Android would include `android-components`.

How are you going to manage a locale translating only Focus, but not Fenix?

Reply:
We are going to have a separate project for Focus, that only contains the Focus specific strings.
Localizers will be directed to Firefox for Android to complete a-c strings if they haven't already (but most locales on Focus have already localized their a-c strings, since most are working on Fenix).
It's a decision we had to make even if it's not ideal. If we add more products in the future, we will talk with Matjaz about possibly doing things differently.